adam/curriculum_to_html.py

```diff
@@ -0,0 +1,184 @@
+import os
+from typing import List
+
+from attr import attrs
+
+from adam.experiment import InstanceGroup
+from adam.language.dependency import LinearizedDependencyTree
+from adam.perception import PerceptualRepresentation
+from adam.perception.developmental_primitive_perception import (
+    DevelopmentalPrimitivePerceptionFrame,
+)
+from adam.situation.high_level_semantics_situation import HighLevelSemanticsSituation
```

Comment:
Add a `__main__` to this class which applies it to the `GAILA_PHASE_1_CURRICULUM`. It should take one argument, a parameter file (using [`Parameters`](https://github.com/isi-vista/vistautils/blob/master/vistautils/parameters.py) from `vistautils`) which specified the output directory.